extension/modules/log4moz.js

```diff
@@ -198,7 +198,9 @@ Logger.prototype = {
   },
 
   _parent: null,
-  get parent() this._parent,
+  get parent() {
+    this._parent;
+  },
```

Comment:
I don't think we need to support gecko <22. We need to make sure it runs with the latest ESR version, but that's 45.

Getters of course don't have a `return`, but I think the could be cleaner if we used arrow functions.